keps/sig-instrumentation/0031-kubernetes-metrics-overhaul.md

```diff
@@ -96,6 +96,10 @@ https://github.com/kubernetes/kubernetes/issues/68522
 
 https://github.com/kubernetes/kubernetes/pull/63924
 
+#### Prober metrics
+
+Make proper metrics introduced in https://github.com/kubernetes/kubernetes/pull/61369 conform to the Kubernetes instrumentation guidelines.
```

Comment:
```suggestion
Make prober metrics introduced in https://github.com/kubernetes/kubernetes/pull/61369 conform to the Kubernetes instrumentation guidelines.
```

Reply:
@brancz -- could you link out to the instrumentation guidelines here as well?

Reply:
done